legend-sdlc-server/src/test/java/org/finos/legend/sdlc/server/gitlab/api/GitLabProjectConfigurationApiTestResource.java

```diff
@@ -76,8 +73,6 @@ public void runUserAndGroupWorkspaceProjectConfigurationTest()
         ProjectConfiguration projectConfiguration = gitLabProjectConfigurationApi.getUserWorkspaceProjectConfiguration(projectId, workspaceOneId);
 
         Assert.assertNotNull(projectConfiguration);
-        Assert.assertTrue(projectConfiguration.getProjectId().startsWith("PROD-"));
```

Comment:
Maybe, this requires us to configure the `projectIdPrefix`, which is configured outside the context of this class, but the assertion is in this method, this will require me to update both `TestGitLabServerProjectConfigurationApis` and `IntegrationTestGitLabProjectConfigurationApis`, I'm not sure if we should do that or we could take that the unit test already taken care of this in a way?